src/src.pro

```diff
@@ -57,6 +57,9 @@ CONFIG(release, debug|release) {
 # VERSION DEFINES
 include(../version.pri)
 
+# Translations
+include(lang/lang.pri)
```

Comment:
A bit offtopic, after we moved this `include()` here, I noticed Qt is building translations multiple times when I run `qmake src.pro` in windows.
I suspect the qmake needs to generate different configuration (debug, release) and built the translations when doing so, so maybe we should move it back to qbittorrent.pro ?

Reply:
>so maybe we should move it back to qbittorrent.pro ?

Yes. At least it should fix this regression, isn't?

This extra delay required to generate .qm files every time I build a project is very annoying to me personally. It would be better to completely get rid of this "intrusive service", whose convenience is very doubtful. It would be better to have some (very simple) script to generate these files so the developer can run it when it is really needed. What do you say? @sledgehammer999?

Reply:
>Yes. At least it should fix this regression, isn't?

yeah, IMO it is the baseline we should do if the situation cannot be improved.

>It would be better to have some (very simple) script to generate these files so the developer can run it when it is really needed. 

I've thought about it, I find it very hard to have a portable script on all platforms/OSs without adding additional build dependency (python, cmake).

Reply:
>It would be better to have some (very simple) script to generate these files so the developer can run it when it is really needed. What do you say? @sledgehammer999?

IMO, it will increase the probability that I'll forget to run it before a release and have stale translations.
Although I suspect it would be possible to intergrate it as a step inside `make tarball` (<--- current way of making release tarballs). But that means we will distribute .qm files too now on each release.